server.go

```diff
@@ -99,6 +106,23 @@ type Server struct {
 	enableProfiling bool
 
 	HistogramAggregates samplers.HistogramAggregates
+
+	// TODO we don't want to hardcode this
+	// TODO we need to actually configure lightstep
+	// TODO also accessing it is a race condition
+	tracerSinks []tracerSink
```

Comment:
Good eye! The first and second TODOs are obsolete (already done). The third isn't really a big deal - it's unexported, and internally we only ever write to it once, so it's not actually problematic.